src/Shaolinq.Postgres/PostgresSqlDatabaseContext.cs

```diff
@@ -51,6 +51,7 @@ protected PostgresSqlDatabaseContext(DataAccessModel model, SqlDialect sqlDialec
 				MaxPoolSize = contextInfo.MaxPoolSize,
 				KeepAlive = contextInfo.KeepAlive,
 				ConnectionIdleLifetime = contextInfo.ConnectionIdleLifetime,
+				ConnectionPruningInterval = contextInfo.ConnectionPruningInterval,

```

Comment:
`ConnectionIdleLifetime` and `ConnectionPruningInterval` both default to `3` seconds in Shaolinq, but the Npgsql defaults are `300` and `10` respectively - any particular reason for the relatively short default timings?